docs/docfiles/user_guides/QuickStart.rst

```diff
@@ -0,0 +1,257 @@
+
+
+.. note::
+
+    The ``dpctl.lsplatform()`` function is new in dpctl 0.7 and will be
+    available in oneAPI 2021.3. If you are following the guide on an older
+    oneAPI installation, use ``dpctl.dump()``. If no GPU platforms are shown,
+    make source your system has a GPU and has the necessary GPU drivers. You can
+    install GPU drivers by following the `GPU driver installation guide`_.
+
+Install Wheel package from Pypi
+-------------------------------
+
+1. Install dpctl
+
+.. code-block:: bash
+
+    python -m pip install --index-url https://pypi.anaconda.org/intel/simple -extra-index-url https://pypi.org/simple dpctl
+
+.. note::
+
+    Dpctl wheel package is placed on Pypi, but some of its dependencies
+    (like Intel numpy) are available only in Anaconda Cloud. That is why install
+    command requires additional intel Pypi channel from Anaconda Cloud.
+
+2. Set path to Performance Libraries in case of using ``venv`` or system Python.
+
+
+On Linux
+
+.. code-block:: bash
+
+    export LD_LIBRARY_PATH=<path_to_your_env>/lib
+
+On Windows
+
+.. code-block:: bat
+
+    set PATH=<path_to_your_env>\bin;<path_to_your_env>\Library\bin;%PATH%
+
+Building from source
+--------------------
+
+To build dpctl from source, we need dpcpp and GPU drivers (and optionally CPU
+OpenCL drivers). It is preferable to use the dpcpp compiler packaged as part of
+oneAPI. However, it is possible to use a custom build of dpcpp and use that to
+build dpctl, especially if you want to enable CUDA support.
+
+Building using oneAPI dpcpp
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As before, oneAPI and graphics drivers should be installed on the system prior
+to proceeding further.
+
+**Activate oneAPI as follows**
+
+On Linux
+
+.. code-block:: bash
+
+  source ${ONEAPI_ROOT}/setvars.sh
+
+On Windows
+
+.. code-block:: bat
+
+    call "%ONEAPI_ROOT%\setvars.bat"
+
+**Build and install using conda-build**
+
+The conda-recipe included with the sources can be used to build the dpctl
+package. The advantage of this approach is that all dependencies are pulled in
+from oneAPI's intelpython conda channel that was installed as part of oneAPI.
+
+.. code-block:: bash
+
+    export ONEAPI_ROOT=/opt/intel/oneapi
+    conda build conda-recipe -c ${ONEAPI_ROOT}/conda_channel
+
+On Windows to cope with `long file names <https://github.com/IntelPython/dpctl/issues/15>`_,
+use ``croot`` with short folder path:
+
+.. code-block:: bat
+
+    set "ONEAPI_ROOT=C:\Program Files (x86)\Intel\oneAPI\"
+    conda build --croot=C:/tmp conda-recipe -c "%ONEAPI_ROOT%\conda_channel"
+
+After building the conda package you may install it by executing:
+
+.. code-block:: bash
+
+    conda install dpctl
+
+.. note::
+
+    You could face issues with conda-build version 3.20. Use conda-build
+    3.18 instead.
+
+**Build and Install with setuptools**
+
+To build using Python ``setuptools``, the following packages should be
+installed:
+
+    - ``cython``
```

Comment:
Let us provide the specific versions of each of these dependencies if not a range of versions

Reply:
I will add a requirements.txt to our repo and reference that in the quick start guide.